schemas/Account.json

```diff
@@ -39,6 +39,10 @@
     "ledger": {
       "description": "A link the the account's ledger",
       "type": "Iri.json"
+    },
+    "fingerprint": {
+      "description": "A fingerprint of the account's client certificate",
+      "type": "string"
```

Comment:
Should we tighten it up with a pattern? Something like: `/^[0-9A-F]{2}(:[0-9A-F]{2}){19}$/`